src/test/enginesynctest.cpp

```diff
@@ -409,10 +411,10 @@ TEST_F(EngineSyncTest, SetExplicitMasterByLights) {
     assertIsExplicitMaster(m_sGroup1);
     assertIsFollower(m_sGroup2);
 
-    // Now set channel 1 to not-master, internal will be master.
+    // Now set channel 1 to not-master, all wil be follower waiting for a valid bpm.
```

Comment:
can this test be updated such that these tracks have bpms?  If you'd like to create a test for the situation where no track has a bpm it should be a different test.  Also, what happens when all tracks are followers and there's no master?

Reply:
Ok, both tests are in place now.